cirq-core/cirq/transformers/transformer_primitives.py

```diff
@@ -275,8 +286,16 @@ def apply_merge_func(op1: ops.Operation, op2: ops.Operation) -> Optional[ops.Ope
                     ).with_tags(*op.tags, _circuit_op_tag)
                 )
                 continue
+
+            op_ck = set(op.classical_controls)
```

Comment:
Nit: this variable name is unclear (classical kontrol?) and it's only used in one place - consider removing it.

Reply:
done